pinot-core/src/test/java/org/apache/pinot/queries/ExplainPlanQueriesTest.java

```diff
@@ -505,7 +505,7 @@ public void testSelectAggregate() {
     result1.add(new Object[]{"AGGREGATE_METADATA", 2, 1});
     check(query1, new ResultTable(DATA_SCHEMA, result1));
 
-    String query2 = "EXPLAIN PLAN FOR SELECT min(invertedIndexCol1) FROM testTable";
+    String query2 = "EXPLAIN PLAN FOR SELECT distinctcounthll(invertedIndexCol1) FROM testTable";
```

Comment:
why change the query to be tested? not the explained plan node below?

Reply:
The purpose of this test is to exercise the plan node; the particular query is unimportant

Reply:
reverted